test/common.js

```diff
@@ -20,6 +20,11 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+// Create tmpDir if it does not exist
+if (! fs.existsSync(exports.tmpDir)) {
+  fs.mkdirSync(exports.tmpDir);
+}
```

Comment:
Hmmm...that will require a `try`/`catch` block which might have a more substantial affect on performance. Will try it anyway and see...